scripts/build-qemu.sh

```diff
@@ -41,6 +45,26 @@ if [ ! -d "$QEMU_SRC_DIR" ]; then
 		cd $QEMU_SRC_DIR
 		git submodule update --init dtc
 	)
+else
+	(
+		cd $QEMU_SRC_DIR
+
+		# Add the remote if it doesn't exist
+		CURRENT_QEMU_REMOTE_NAME=$(git remote -v | grep fetch | grep "$QEMU_REMOTE_BIT" | sed -e's/\t.*$//')
+		if [ x"$CURRENT_QEMU_REMOTE_NAME" = x ]; then
+			git remote add $QEMU_REMOTE_NAME $QEMU_REMOTE
+			CURRENT_QEMU_REMOTE_NAME=$QEMU_REMOTE_NAME
+		fi
+
+		# Get any new data
+		git fetch $CURRENT_QEMU_REMOTE_NAME
+
+		# Checkout or1k-linux branch it not already on it
```

Comment:
From context it looks like this comment needs updating for `scripts/build-qemu.sh`: it appears to be checking out "master" branch here, not `or1k-linux`.  Rest of the change looks plausible though.